akka-remote/src/main/resources/reference.conf

```diff
@@ -108,36 +108,69 @@ akka {
       # stream, to pass such messages through the large message stream the selections
       # but must be resolved to ActorRefs first.
       large-message-destinations = []
-      
+
       advanced {
         # For enabling testing features, such as blackhole in akka-remote-testkit.
         test-mode = off
-      
+
         # Settings for the materializer that is used for the remote streams.
         materializer = ${akka.stream.materializer}
         materializer {
           dispatcher = "akka.remote.default-remote-dispatcher"
         }
-        
+
         # Controls whether to start the Aeron media driver in the same JVM or use external
-        # process. Set to 'off' when using external media driver, and then also set the 
+        # process. Set to 'off' when using external media driver, and then also set the
         # 'aeron-dir'.
         embedded-media-driver = on
-        
+
         # Directory used by the Aeron media driver. It's mandatory to define the 'aeron-dir'
         # if using external media driver, i.e. when 'embedded-media-driver = off'.
         # Embedded media driver will use a this directory, or a temporary directory if this
         # property is not defined (empty).
         aeron-dir = ""
-        
+
+        # Whether to delete aeron embeded driver directory upon driver stop.
+        delete-aeron-dir = yes
+
         # Level of CPU time used, on a scale between 1 and 10, during backoff/idle.
         # The tradeoff is that to have low latency more CPU time must be used to be
         # able to react quickly on incoming messages or send as fast as possible after
-        # backoff backpressure. 
+        # backoff backpressure.
         # Level 1 strongly prefer low CPU consumption over low latency.
-        # Level 10 strongly prefer low latency over low CPU consumption. 
+        # Level 10 strongly prefer low latency over low CPU consumption.
         idle-cpu-level = 5
-        
+
+        system-message-resend-interval = 1 second
+        handshake-retry-interval = 1 second
+        inject-handshake-interval = 1 second
+        give-up-send-after = 60 seconds
+        shutdown-flush-timeout = 1 second
```

Comment:
`# during ActorSystem termination the remoting will wait this long for a acknowledgment by the destination system that flushing of outstanding remote messages has been completed`